src/Raven.Server/Rachis/Leader.cs

```diff
@@ -817,6 +819,19 @@ public void Dispose()
             }
         }
 
+        void ObserveTask(Task task)
```

Comment:
better to use the `IgnoreUnobservedExceptions` extension

Reply:
Done